CHANGELOG.md

```diff
@@ -2,11 +2,12 @@
 
 ## 4.10.1
 
+### Features
```

Comment:
@krystofwoldrich make it to `Unreleased`

Reply:
On it. :)